next-docs/pages/world/balance.mdx

```diff
@@ -1 +1,15 @@
 # Balance
+
+ETH is stored in the `World` contract and owned by namespaces.
+Any ETH sent with a call to a `System` is kept on the `World`, and credited to the namescape to which the `System` belongs.
+This has two important effects:
+
+- The ETH balance is shared between the different `System` contracts in the same namespace of the same `World`.
+- The same `System` can be used in multiple `World` contracts, and in multiple namespaces in the same `World`, and the ETH balances remain separate.
```

Comment:
```suggestion
- The same `System` can be used in multiple `World` contracts and the ETH balances remain separate.
```
A system contract actually can't be registered multiple times in the same world. This has to do with the requirement to have a 1-1 mapping between system address and system ID. (Let's make a note about that on the Systems doc page.

Reply:
this is still relevant